File: Session_01_InputAndOutput/cpp/basics.cpp
#include <iostream>
#include <string>

int main() {
    // 1. Output
    std::cout << "Hello from C++!" << std::endl;

    // 2. Variables
    int age = 19;
    double gpa = 3.75;
    char grade = 'A';
    std::string name = "Budi";

    std::cout << "Name: " << name << std::endl;
    std::cout << "Age: " << age << " | GPA: " << gpa << " | Grade: " << grade << std::endl;

    // 3. Input
    std::cout << "Enter a new age: ";
    std::cin >> age;

    std::cout << "Updated Age: " << age << std::endl;

    return 0;
}

File: Session_01_InputAndOutput/python/basics.py
# 1. Output
print("Hello from Python!")

# 2. Variables (dynamically typed)
age = 19
gpa = 3.75
grade = 'A'
name = "Budi"

print(f"Name: {name}")
print(f"Age: {age} | GPA: {gpa} | Grade: {grade}")

# 3. Input
# Note: input() always returns a string, so we must cast it to int if needed
age_input = input("Enter a new age: ")
age = int(age_input)

print(f"Updated Age: {age}")
